Dockerfile

```diff
@@ -8,6 +8,11 @@ EXPOSE ${PRESTO_PORT}
 WORKDIR /app
 ENV DEBIAN_FRONTEND=noninteractive
 
+RUN git clone https://github.com/facebookresearch/sscd-copy-detection.git
```

Comment:
maybe these should be installed after the system library stuff?

Reply:
I've solved the build issues and moved the needed packages to `requirements.txt` where they belong @ahmednasserswe